bugbug/models/duplicate.py

```diff
@@ -94,10 +97,17 @@ def get_labels(self):
                 if duplicate_bug_id not in all_ids:
                     continue
 
+                local_duplicates.append(duplicate_bug_id)
                 duplicate_ids.append(duplicate_bug_id)
 
                 if duplicates_num < self.num_duplicates:
                     classes[(bug_id, duplicate_bug_id)] = 1
```

Comment:
Do we still need this now that we have the for loop?

Reply:
Or is this particular tuple already set to 1 in the for loop?

Reply:
I guess it isn't right now, but you could do that simply by initializing `local_duplicates` this way:
```
local_duplicates = [bug_id]
```

Reply:
We already have `(bug_id, duplicate_bug_id)` = 1. We need to add all pairs of duplicate_bug_ids to this set. Hence, I'm filling local_duplicates with only the duplicate_ids and using itertools.combinations to generate more labels. Is this not right?

Reply:
Yeah, I was just suggesting to have:
```
local_duplicates = [bug_id]

...

if duplicates_num < self.num_duplicates:
    for duplicate_pair in combinations(local_duplicates, 2):
        classes[tuple(duplicate_pair)] = 1
        duplicates_num += 1
```
as an alternative implementation, so that you can remove the `classes[(bug_id, duplicate_bug_id)] = 1` line.

Reply:
Ah, I see. Will do 👍